### ISSUE TITLE
Delay while building

### ISSUE THREAD

**Issue by ecksun** [fuzzylite/fuzzylite#58]
I'm curious as to why we are waiting 3 seconds before we start the build?
Tt doesn't seem to serve any purpose and only slows down the development cycle.

**Comment by jcrada** [comment]
Hi, thank you for your post. 

The three seconds are to allow time to cancel the build if necessary. The script files are only to facilitate the building procedure, but you can always just use CMake directly and have more control over the building process. I will try to add an option to the script to control the sleeping time, though.

**Comment by ecksun** [comment]
Alright, sure :P

I'm currently working on a node.js module that uses the this library and part of that requires building the code. I'm just running `build.sh` since thats whats documented as the build procedure. I would prefer to build with default-arguments to reduce the risk of diverging from this repository's build-system. With that in mind, I think it might be beneficial to either document the cmake arguments thoroughly and/or make sure sane defaults are used.

It might be that they both are documented and that sane defaults currently are configured, but in that case I don't know where to find that information (which then in itself might be worth documenting).

Perhaps this discussion is a bit off topic though, if you think so I will create a new issue for it.

**Comment by jcrada** [comment]
Please, check the CMake command in `build.sh`. Information about the arguments can be found here: http://www.fuzzylite.com/cpp/#Building_from_source . You can also look at the CMakeLists.txt for specific information.

**Comment by ecksun** [comment]
Ah, yes, so I just missed it, good :)

Then I will close this issue, thanks for the help.
